app/views/topics/show.html.erb

```diff
@@ -32,7 +32,8 @@
         <div class="document-type-group-list">
           <h3><%= document_type.humanize %>:</h3>
           <ul>
-            <% content_items.each do |item| %>
+            <% content_items.each_with_index do |item, index| %>
+              <% break if index > 4 %>
```

Comment:
I think the logic can be simplified here by using the [`take` method](http://ruby-doc.org/core-1.9.2/Enumerable.html#method-i-take). E.g:

`content_items.take(5).each_with_index do |item, index|`

Reply:
I have never seen this before. Good to know. Thanks I will implement this. 💪 👨‍💻